scripts/setUpSite.js

```diff
@@ -15,17 +14,40 @@ const base = new TileLayer(
   }
 );
 
+const setUpCityPointsLayer = async (map) => {
+  const data = await import("../map/tidied_map_data.csv");
+  const scope = {
+    Regional: "#7b3294",
+    "City Center": "#fdae61",
+    Citywide: "#d7191c",
+    "Main Street": "#abdda4",
+    TOD: "#2b83ba",
+  };
+  data.forEach((mandate) => {
+    new CircleMarker([mandate.lat, mandate.long], {
+      radius: 7,
+      stroke: true,
+      weight: 0.9,
+      color: "#FFFFFF",
+      fillColor: scope[mandate.magnitude_encoded],
+      fillOpacity: 1,
+    }).addTo(map);
+  });
+};
+
 const setUpSite = async () => {
   setUpIcons();
 
   const map = new Map("map", {
     layers: [base],
   });
-  map.setView([40.9437, -78.9709], 12); // latitude, longitude
+  map.setView([43.2796758, -96.7449732], 4); // Set default view (lat, long) to United States
 
   map.attributionControl.setPrefix(
     'created by <a style="padding: 0 3px 0 3px; color:#fafafa; background-color: #21ccb9;" href=http://www.geocadder.bg/en/>GEOCADDER</a>'
   );
+
+  await Promise.all([setUpCityPointsLayer(map)]);
```

Comment:
Let's keep it simple for now. We should only introduce `Promise.all` when we have multiple promises and we are confident the two don't depend on each other.

```suggestion
  await setUpCityPointsLayer(map);
```